cmd/main.go

```diff
@@ -50,45 +50,20 @@ func main() {
 	http.Handle("/metrics", promhttp.Handler())
 	go http.ListenAndServe(":9080", nil)
 
-	// Install iptables rule to masquerade IPv4 NAT64 traffic
-	ipt4, err := iptables.NewWithProtocol(iptables.ProtocolIPv4)
-	if err == nil {
-		klog.Infof("Running on IPv4 mode")
-		networkPolicyController4 := networkpolicy.NewController(
-			clientset,
-			informersFactory.Networking().V1().NetworkPolicies(),
-			informersFactory.Core().V1().Namespaces(),
-			informersFactory.Core().V1().Pods(),
-			ipt4,
-			104,
-		)
-		go networkPolicyController4.Run(ctx, 5)
-	} else {
-		klog.Infof("Error running on IPv4 mode: %v", err)
-	}
-
-	/* TODO make this configurable, it can be ipv4, ipv6 or dual
-	ipt6, err := iptables.NewWithProtocol(iptables.ProtocolIPv6)
+	nft, err := knftables.New(knftables.InetFamily, "kube-netpol")
 	if err != nil {
-		log.Fatalf("Could not use iptables IPv6: %v", err)
+		klog.Fatalf("Error initializing nftables: %v", err)
 	}
-	*/
 
-	ipt6, err := iptables.NewWithProtocol(iptables.ProtocolIPv6)
-	if err == nil {
-		klog.Infof("Running on IPv6 mode")
-		networkPolicyController6 := networkpolicy.NewController(
-			clientset,
-			informersFactory.Networking().V1().NetworkPolicies(),
-			informersFactory.Core().V1().Namespaces(),
-			informersFactory.Core().V1().Pods(),
-			ipt6,
-			106,
-		)
-		go networkPolicyController6.Run(ctx, 5)
-	} else {
-		klog.Infof("Error running on IPv6 mode: %v", err)
-	}
+	networkPolicyController := networkpolicy.NewController(
+		clientset,
+		informersFactory.Networking().V1().NetworkPolicies(),
+		informersFactory.Core().V1().Namespaces(),
+		informersFactory.Core().V1().Pods(),
+		nft,
+		104,
```

Comment:
TODO this should be a constant or something, if we need to do multiqueue later https://home.regit.org/netfilter-en/using-nfqueue-and-libnetfilter_queue/ 

```
iptables -A INPUT -j NFQUEUE --queue-balance 0:3
```